cmd/generate/config/rules/openstack.go

```diff
@@ -0,0 +1,26 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+	"regexp"
+)
+
+func OpenStackRCPassword() *config.Rule {
+	// define rule
+	r := config.Rule{
+		RuleID:      "openstack_rc_password",
+		Description: "OpenStack RC file password",
+		Regex:       regexp.MustCompile(`export OS_PASSWORD=(.*)`),
```

Comment:
@srelf-eschercloud can you restrict the secret group? I don't accept (.*). https://blog.gitleaks.io/finding-secrets-with-regular-expressions-d90493bb3784 might give you some ideas

Reply:
sure ill take a look.